common/djangoapps/student/models.py

```diff
@@ -919,7 +920,16 @@ class CourseEnrollment(models.Model):
     MODEL_TAGS = ['course_id', 'is_active', 'mode']
 
     user = models.ForeignKey(User)
-    course_id = CourseKeyField(max_length=255, db_index=True)
+    batch_course_overview = models.ForeignObject(
+        CourseOverview,
+        from_fields=['course_id'],
+        to_fields=['id'],
+        blank=True,
+        null=True,
+        related_name='+',
+        on_delete=DO_NOTHING
+    )
```

Comment:
Haha, shit.  Didn't even think of that.  I'll see what `makemigrations` thinks about it.